deeplearning4j-nn/src/main/java/org/deeplearning4j/util/ModelSerializer.java

```diff
@@ -168,7 +167,7 @@ public static void writeModel(@NonNull Model model, @NonNull OutputStream stream
             NormalizerSerializer.getDefault().write(dataNormalization, zipfile);
         }
 
-
+        dos.close();
```

Comment:
Please, move closing resources into `finally` statement to make sure that resources will be closed anyway.